ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<blockquote>train/test/validate splits (global comment)</blockquote><p>In the docs, we refer to this as "data splits" (https://cloud.google.com/vertex-ai/docs/general/ml-use). Might be easier to read than "train/test/validate splits"? The doc link also includes a description of the default data splits.</p><p> </p><blockquote>You must wait for the training process to complete before you can get the resource name and ID of your new model.</blockquote><p>Suggest adding: "..., which is required for model evaluation and model deployment." to explain why this is required</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Done